app/models/interactive_page.rb

```diff
@@ -375,19 +377,46 @@ def self.import(page_json_object, helper=nil)
         end
       end
 
-      page_json_object[:sections].each do |s|
+      # if explicit version not set figure it out from the json
+      version = version || (page_json_object.has_key?(:sections) ? 2 : 1)
+
+      if version == 1 
         # First, import and cache all the embeddables.
-        s[:embeddables].each do |embed_json_obj|
-          embed = helper.import(embed_json_obj.except(:column, :position))
-          import_page.add_embeddable(embed, embed_json_obj[:position], s[:title], embed_json_obj[:column])
+        page_json_object[:embeddables].each do |embed_hash|
+          embed = helper.import(embed_hash[:embeddable])
+          section = embed_hash[:section]
+          import_page.add_embeddable(embed, nil, section)
         end
 
         # Now when all the objects are created, setup references (e.g. question pointing to interactive, or
         # one embeddable pointing to another one).
-        s[:embeddables].each do |embed_json_obj|
-          helper.set_references(embed_json_obj)
+        page_json_object[:embeddables].each do |embed_hash|
+          helper.set_references(embed_hash[:embeddable])
+        end
+      elsif version == 2
+        page_json_object[:sections].each do |s|
+          # first create the section and add it to the page at the correct location
+          new_section = import_page.sections.create(Section::DEFAULT_PARAMS.merge({title: s[:title]}))
+          if s[:title] == Section::HEADER_BLOCK
+            new_section.move_to_top
```

Comment:
I'd recommend removing this. If we are importing a version 2 page, the sections should be ordered correctly in the json we are parsing. The titles shouldn't have any effect on the order in a version 2 page.

Reply:
I'll remove it.

Reply:
Removed in https://github.com/concord-consortium/lara/pull/1008/commits/b788907652229ee237501d1131876db8aac31082